### ISSUE TITLE
Bomb edge conditions

### ISSUE THREAD

**Issue by suphoff** [MultiAgentLearning/playground#77]
There are a few minor edge conditions concerning bombs 
(Just verified the first item with player actors - did not verify the other item)

- If agent X drops a bomb on position P the board shows agent X on P. If Agent X steps off position P it allows another Agent Y to step on the bomb on the same turn. (Kick will not be delivered and kick ability not required)
-  The check if a bomb can be kicked utilizes mostly board information from the last step (only flame out and revealed items are updated). This can cause bomb movement on allowed kicks to be immediately blocked by unrelated agents  - causing the bomb to stop and the kicking agent to step on the bomb. Kicks may also be disallowed by agents that used to be in a blocking position but moved off that position.
- Not allowed to kick bomb into flames or onto items (not sure if that is desired or not)

**Comment by cinjon** [comment]
Addressed the 1st and the third issues in this commit --> https://github.com/MultiAgentLearning/playground/commit/b28a47740e24ab009466ed5a7b77c61439c06bda. The 2nd one might take some more to fix. Do you mind submitting another issue for that one (and/or fixing it with a PR)? Thanks again @suphoff !
